pyproject.toml

```diff
@@ -40,6 +40,7 @@ pytest-mockservers = "^0.6.0"
 pytest-sugar = "^0.9.4"
 time-machine = "^2.7.0"
 yamllint = "^1.26.3"
+types-requests = "^2.31.0.6"
```

Comment:
This goes in `tool.poetry.dependencies` not the `dev` group.

Reply:
Change has done :)